overlay/sbin/toolbox

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Licened under Apache 2.0
+# A simple way to bring your own tools to the K3OS Kubernetes Linux distribution
+# Inspired by https://github.com/containers/toolbox
+
+if [ $(whoami) != "root" ]
+then
+	echo "This script must be run as root."
+	exit 1
+fi
+
+if [ "$1" = "" ]; then
+	echo "toolbox: missing command" >&2
+	echo >&2
+	echo "These are some common commands:" >&2
+	echo "create	Create a new toolbox container" >&2
+	echo "enter	 Enter an existing toolbox container" >&2
+	echo "rm	 Remove an existing toolbox container" >&2
+	echo >&2
+	exit 1
+fi
+
+op=$1
+shift
+
+CTR='k3s ctr'
+registry="docker.io/library"
+image="ubuntu:19.10"
+
+case $op in
+	create )
+		# Check if the container already exists
+		# If not, create it.
+		${CTR} container info toolbox &> /dev/null
+		INFO=$(echo $?)
+		if [ ${INFO} == "1" ]
+		then
+			${CTR} image pull ${registry}/${image}
+			${CTR} run -d --net-host --privileged ${registry}/${image} toolbox bash
+		else
+			echo "Container 'toolbox' already exists"
+			exit 1
+		fi
+		;;
+	enter )
+		${CTR} task exec -t --exec-id 100 toolbox bash
```

Comment:
We are explicitly depending on `bash` being available, which is fine, but some preferred containers (Alpine seems a likely candidate) do not have `bash` by default. We should revert to `sh` or provide some helpful error-condition text indicated something like "Does your preferred container have bash installed?"

Reply:
That's a good call, I don't want to break folks' Alpine. Adopted with ability to override in favor of bash.